e2e/js/src/js-generator-publishable.test.ts

```diff
@@ -0,0 +1,86 @@
+import { getPackageManagerCommand } from '@nx/devkit';
+import {
+  cleanupProject,
+  getSelectedPackageManager,
+  newProject,
+  readJson,
+  runCLI,
+  uniq,
+  updateJson,
+} from '../../utils';
+
+describe('js:generators:publishable', () => {
+  let scope: string;
+  let dlxCommand: string;
+
+  beforeAll(() => {
+    scope = newProject({
+      name: uniq('js_generators_publishable'),
+      packages: ['@nx/js'],
+    });
+    dlxCommand = getPackageManagerCommand(getSelectedPackageManager()).dlx;
+  });
+
+  afterAll(() => cleanupProject());
+
+  it('should not add projects config when it does not exist', async () => {
```

Comment:
This would run quicker if it was a unit test, and none of the functionality requires a full workspace. We should consider changing it.

Reply:
This logic is covered by unit tests as well (in library.spec.ts), so I'll just delete this e2e